src/components/PopularList.jsx

```diff
@@ -0,0 +1,47 @@
+import '../styling/PopularList.css'
+import { Link } from 'react-router-dom'
+
+//functional component
+export const PopularList = ({ movieData }) => {
+	console.log('moviedata popularLIst', movieData)
+
+	return (
+			<div className="movie-wrapper">
+				<div className="movie-list">
+			{movieData &&
+				movieData.results.map((movie) => (
+					<Link key={movie.id} to={`/details/${movie.id}`}>
+								<div className="movie-card" key={movie.id}>
+									<div className="hover-text">
+										<h1 className="title">{movie.title}</h1>
+										<h2 className="release-date">
+											Released {movie.release_date}
+										</h2>
+									</div>
+									<div className="poster">
+										<img
+											className="movie-image"
+											src={`https://image.tmdb.org/t/p/w342${movie.poster_path}`}
+											alt={`Movie poster for ${movie.title}`}
+										/>
+									</div>
+								</div>
+					</Link>
```

Comment:
Code looks great but it gets a bit complicated to read due to the unorganised set up. 
-Izabel&Etna